{
  "exploitdb": [],
  "itw": [
    {
      "id": "CVE-2020-11048",
      "description": "In FreeRDP after 1.0 and before 2.0.0, there is an out-of-bounds read. It only allows to abort a session. No data extraction is possible. This has been fixed in 2.0.0.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2020-11048",
      "reportURL": "https://github.com/FreeRDP/FreeRDP/issues/6007",
      "timeStamp": "2020-08-30T02:15:00Z"
    }
  ],
  "trickest": [],
  "nomisec": [],
  "nuclei": [],
  "metasploit": [],
  "holloways": [],
  "openpoc": [
    {
      "cve": "CVE-2020-11048",
      "url": "https://github.com/FreeRDP/FreeRDP/issues/6007",
      "added_at": "2020-08-30T02:15:00Z",
      "score": 0.4,
      "template_for": ""
    }
  ]
}
